.pre-commit-config_template.yaml

```diff
@@ -7,10 +7,13 @@ repos:
   - id: check-json
   - id: check-yaml
   - id: check-ast
+    min_py_version: "3.7"
```

Comment:
can we use a constant? this value is repeated multiple times

Reply:
I don't want to overcomplicte this YAML with anchors.
In addition, when we upgrade the hook versions we will need to check this value, as it's possible that the newer version dropped support (for example, ruff supports now python 3.8 and up)